src/main/config/messages_template.xml

```diff
@@ -7,413 +7,415 @@ Copyright 2005, 2006 IBM Corporation
     <response></response>
   </message>
 
   <message id="DOTJ069E" type="ERROR">
-    <reason>Circular key definition %1.</reason>
+    <reason>Circular key definition '%1'.</reason>
     <response></response>
   </message>
 
   <message id="DOTJ070I" type="INFO">
-    <reason>Invalid 'class' attribute '%1' for was found for element '&lt;%2&gt;'.</reason>
-    <response>The element will be processed as an unknown or non-DITA element.</response>
+    <reason>An invalid @class attribute '%1' was found for the &lt;%2&gt; element.</reason>
+    <response>Processing as an unknown or non-DITA element.</response>
   </message>
 
   <message id="DOTJ071E" type="ERROR">
-    <reason>Cannot find the specified DITAVAL '%1'.</reason>
+    <reason>Cannot find the specified DITAVAL file '%1'.</reason>
     <response></response>
   </message>
 
   <message id="DOTJ072E" type="ERROR">
-    <reason>Email link without correct 'format' attribute.</reason>
-    <response>Using 'format' attribute value 'email'.</response>
+    <reason>Email link without correct @format attribute.</reason>
+    <response>Using @format attribute value 'email'.</response>
   </message>
 
   <message id="DOTJ073E" type="ERROR">
-    <reason>Email link without correct 'scope' attribute.</reason>
-    <response>Using 'scope' attribute value 'external'.</response>
+    <reason>Email link without correct @scope attribute.</reason>
+    <response>Using @scope attribute value 'external'.</response>
   </message>
 
   <message id="DOTJ074W" type="WARN">
-    <reason>Rev attribute cannot be used with prop filter.</reason>
+    <reason>The @rev attribute cannot be used with the &lt;prop&gt; filter.</reason>
     <response></response>
   </message>
 
   <message id="DOTJ075W" type="WARN">
-    <reason>Absolute link '%1' without correct 'scope' attribute.</reason>
-    <response>Using 'scope' attribute value 'external'.</response>
+    <reason>Absolute link '%1' without correct @scope attribute.</reason>
+    <response>Using @scope attribute value 'external'.</response>
   </message>
 
   <message id="DOTJ076W" type="WARN">
-    <reason>Absolute link '%1' without correct 'scope' attribute.</reason>
+    <reason>Absolute link '%1' without correct @scope attribute.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTJ077F" type="FATAL">
-    <reason>Invalid action attribute '%1' on DITAVAL property.</reason>
+    <reason>Invalid @action attribute '%1' on DITAVAL property.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTJ078F" type="FATAL">
-    <reason>Input file '%1' could not be loaded.</reason>
-    <response>Ensure that grammar files for this document type are referenced and installed properly.</response>
+    <reason>The input resource '%1' cannot be loaded.</reason>
+    <response>Make sure the grammar files for this document type are properly referenced and installed.</response>
```

Comment:
One spot above, we used "DTD or schema files", here we use "grammar files" ... honestly not sure which is better / more understandable or if it's worth updating. I think "grammar files" is more precise / accurate, but people might be more inclined to understand "DTD or schema"?

Reply:
We can use both: _“…grammar files (such as DTDs or schemas)…”_